\doxysection{pip.\+\_\+internal.\+exceptions.\+Bad\+Command Class Reference}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1_bad_command}{}\label{classpip_1_1__internal_1_1exceptions_1_1_bad_command}\index{pip.\_internal.exceptions.BadCommand@{pip.\_internal.exceptions.BadCommand}}
Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Bad\+Command\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=3.000000cm]{classpip_1_1__internal_1_1exceptions_1_1_bad_command}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when virtualenv or a command is not found\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
C\+:/\+Users/kmarty/\+Documents/\+Labo3\+\_\+architecture\+\_\+killian\+\_\+marty/\+Lib/site-\/packages/pip/\+\_\+internal/exceptions.\+py\end{DoxyCompactItemize}
